BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -106,7 +106,12 @@ public BraintreeClient(@NonNull Context context, @NonNull String authorization,
         this.graphQLClient = params.getGraphQLClient();
         this.httpClient = params.getHttpClient();
         this.manifestValidator = params.getManifestValidator();
-        this.sessionId = params.getSessionId();
+
+        String sessionId = params.getSessionId();
+        if (sessionId == null) {
+            sessionId = params.getUuidHelper().getFormattedUUID();
```

Comment:
```suggestion
            sessionId = params.getUUIDHelper().getFormattedUUID();
```
I guess second pass it might make sense to capitalize `UUID` since it's an acronym.

Reply:
Good call - updated this and the setter 👍